Mayden-Academy/2020-jan-officeSignIn
# S3 t3 display visitors route

Added route to retrieve all signed in visitors.
To test:
run phpunit tests and ensure all pass.
- make a GET request to localhost:8080/api/admin and ensure some data comes back (check your database - for this to work you need to have people who are signed in on today's date).
- set everyone in the DB to signed out, then make another GET request, and make sure you receive a message that no data could be retreived.
